### PR TITLE
PLANET-7618 Make sure nav links don't exceed 2 lines

### PR DESCRIPTION
**Pull request by mleray**
### Description

See [PLANET-7618](https://jira.greenpeace.org/browse/PLANET-7618)

This can happen in edge cases where a page used in the navigation is renamed with a longer title

### Testing

On the [titan instance](https://www-dev.greenpeace.org/test-titan/) in the navigation under `The issues we work on` I've added a page with a really long title. It should be cut off after 2 lines for both mobile and desktop.

### CODE REVIEW

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [titan](https://www-dev.greenpeace.org/test-titan) | [admin](https://www-dev.greenpeace.org/test-titan/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-titan/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-titan) | [composer-local.json](https://github.com/greenpeace/planet4-test-titan/blob/main/composer-local.json)

:watch: 2024.11.22 14:22:00

File: assets/src/scss/layout/_navbar.scss [hunk 1]
```diff
@@ -208,6 +208,14 @@ a.nav-link {
   width: 100%;
   text-decoration: none !important;
 
+  span {
+    display: -webkit-box;
+    -webkit-box-orient: vertical;
+    line-clamp: 3;
+    -webkit-line-clamp: 2;
```
Comment (mardelnet) [inline assets/src/scss/layout/_navbar.scss:215]: Shouldn't we use the same value for both properties?

File: assets/src/scss/layout/_navbar.scss [hunk 2]
```diff
@@ -208,6 +208,14 @@ a.nav-link {
   width: 100%;
   text-decoration: none !important;
 
+  span {
```
Comment (mardelnet) [inline assets/src/scss/layout/_navbar.scss:211]: I see we repeat this set of properties many times across the project, what about making them reusable? Perhaps in a different PR.

File: assets/src/scss/layout/_navbar.scss (hunk 1 above)
Comment (mleray) [reply assets/src/scss/layout/_navbar.scss:215]: Absolutely, yes 🤦‍♀️

Comment (mleray) [reply assets/src/scss/layout/_navbar.scss]: Updated!
